src/cloud/components/Modal/contents/Doc/RevisionsModal/RevisionModalDetail.tsx

```diff
@@ -97,23 +99,26 @@ const RevisionModalDetail = ({
           alignItems='baseline'
         >
           <h3>
-            Updated at {format(new Date(revision.created), 'HH:mm, dd MMMM u')}
+            Updated at {format(new Date(revisionCreatedAt), 'HH:mm, dd MMMM u')}
           </h3>
-          <span>
-            {revision.creators.length === 0 ? (
-              <i>unknown</i>
-            ) : (
-              <>
-                by {revision.creators.map((user) => user.displayName).join(',')}
-              </>
-            )}
-          </span>
+          {revisionCreators != null && (
+            <span>
+              {revisionCreators.length === 0 ? (
+                <i>unknown</i>
```

Comment:
Do we want to show unknown?  I feel like it would probably be fine just showing 

```
{revisionCreators != null && revisionCreators.length === 0 ? (
```